src/lib/recipes.ts

```diff
@@ -422,6 +422,104 @@ const xiphosRecipes: IRecipe[] = [
   ),
 ];
 
+// HH-a "Javelin" half-truck recipes
+const javelinRecipes: IRecipe[] = [
```

Comment:
javelin recipes:

Garage | 55 x Refined Materials | HH-a “Javelin” | 55  hits

Mass Production Factory | 395 x Refined Materials | 3 crates  of 3 x HH-a “Javelin” | 02:17:30
Mass Production Factory | 494 x Refined Materials | 4 crates  of 3 x HH-a “Javelin” | 03:03:20
Mass Production Factory | 576 x Refined Materials | 5 crates  of 3 x HH-a “Javelin”